src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/AdminMenu.cs

```diff
@@ -24,7 +24,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             builder
                 .Add(S["Configuration"], "10", design => design
                     .AddClass("menu-configuration").Id("configuration")
```

Comment:
> The difficulty for some menus like Configuration and Search is indeed to find a "main" feature on which all other sub features depend on.

@agriffard agreed, this is the dificulty! so maybe the answers is to do it everywhere? because mostly these are going to be cut and pasted into new menus, so good to get the cut and paste code right.

Also noting that it must be last module registration wins, because here it is `10` and yet `100` is winning